src/ScopeRequest.js

```diff
@@ -102,6 +111,28 @@ class ScopeRequest {
     return true;
   }
 
+  /**
+   * Validate the constraints of an Scope Request
+   * @param filter of an aggregation in the Scope Request
+   * @returns {boolean} true|false
+   */
+  static validateAggregationFilter(filter) {
+    const operatorKeys = _.keys(filter);
+
+    if (operatorKeys.length !== 1) {
+      throw new Error('Invalid Constraint Object - only one operator is allowed');
+    }
+    if (!_.includes(VALID_AGGREGATORS, operatorKeys[0])) {
```

Comment:
if we decide to remove the lengths constraint (which I think we should), then this should all be in a 'forEach' loop and the errors should probably be an array

Reply:
There is no limit, the limit is only that we can do operators like this:

{ operator: value }

But we can do:

[{operator: value}, {operator: value}]

Reply:
ok, I missed that it was just on multiple keys in a block. Looks good then